src/js/app.js

```diff
@@ -32,18 +37,22 @@ const onNetworkReset = () => {
   renderHierarchy(data.hierarchy);
 };
 
-const isNetworkReset = () => {
+const isNodesFiltered = () => {
```

Comment:
Please comment your function definitions

Reply:
Self explanatory. Any further documentation is noise than helpful in this case